package.json

```diff
@@ -103,6 +103,7 @@
     "highlight.js": "9.18.1",
     "intl": "1.2.5",
     "jasminewd2": "2.2.0",
+    "karma-viewport": "1.0.7",
```

Comment:
Just qq, is it needed?

Reply:
Hey @JKMarkowski , as per Fiori 3 guidelines for Responsiveness, the layout should automatically change depending on the screen size. For example, a 'THREE_COLUMNS_MID_EXPANDED' should render 3 columns, start open, mid expanded, end open in a large screen. This layout is changed to 'THREE_COLUMNS_START_MINIMIZED' in MD screen. I need a way to control the screen size in order to test these 2 cases. One, where the screen is large and one, where I pass this layout but the screen is tablet size. "karma-viewport" allows me to do this by setting the size of the viewport while testing. In the test file you will see `viewport.set(1023, 900);`. That's where I specify in which viewport I am testing the layout. 
If you think I should remove these test cases OR you know a better way to do it, please let me know and I will remove "karma-viewport" package.

Reply:
Ok @InnaAtanasova , Thank You for clarification :D We should keep it then. Probably we'll need to test such a ase on other components as well